src/page/address-step.page.ts

```diff
@@ -4,7 +4,7 @@ export class AddressStepPage {
   private proceedToCheckoutButton: ElementFinder;
 
   constructor () {
-    this.proceedToCheckoutButton = $('#center_column > form > p > button > span');
+    this.proceedToCheckoutButton = $('.cart_navigation > .button > span');
```

Comment:
try to find a better locator, something that uses the name or title attribute for example, in that way, we can understand what element you are referring